src/Mod/Sandbox/Gui/Command.cpp

```diff
@@ -1150,13 +1090,6 @@ CmdTestGDIWidget::CmdTestGDIWidget()
 
 void CmdTestGDIWidget::activated(int)
 {
-#ifdef Q_OS_WIN32

```

Comment:
Your changes are OK except for this one. GDIWidget is a test class to see how Qt widgets can be combined with Win32's GDI system. As the name says the Sandbox workbench is only there to play with some exotic features of used libraries.